core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/FileBatchSource.java

```diff
@@ -228,7 +222,7 @@ public FileBatchConfig() {
       super("");
       this.fileSystemProperties = GSON.toJson(ImmutableMap.<String, String>of());
       this.fileRegex = ".*";
-      this.inputFormatClass = CombineTextInputFormat.class.getName();
+      this.inputFormatClass = TextInputFormat.class.getName();
```

Comment:
Any reason behind this change? I guess it should be have TextInputFormat to begin with.